app/javascript/packages/document-capture/components/location-collection-item.tsx

```diff
@@ -46,6 +60,15 @@ function LocationCollectionItem({
         <div>{`${t('in_person_proofing.body.location.retail_hours_weekday')} ${weekdayHours}`}</div>
         <div>{`${t('in_person_proofing.body.location.retail_hours_sat')} ${saturdayHours}`}</div>
         <div>{`${t('in_person_proofing.body.location.retail_hours_sun')} ${sundayHours}`}</div>
+        {phone || tty ? (
+          <div>
+            <h4>{t('in_person_proofing.body.location.contact_info_heading')}</h4>
+            <div>{`${t('in_person_proofing.body.location.phone')} ${phone}`}</div>
+            <div>{`${t('in_person_proofing.body.location.tty')} ${tty}`}</div>
+          </div>
+        ) : (
+          ''
+        )}
```

Comment:
Slight preference for && instead of ternary operator, so you don't have to specify the empty string

```suggestion
        {(phone || tty) && (
          <div>
            <h4>{t('in_person_proofing.body.location.contact_info_heading')}</h4>
            <div>{`${t('in_person_proofing.body.location.phone')} ${phone}`}</div>
            <div>{`${t('in_person_proofing.body.location.tty')} ${tty}`}</div>
          </div>
        )}
```

Reply:
that's definitely a lot neater!